config-templates/linux-local-dev-segue-config.properties

```diff
@@ -112,3 +113,6 @@ BETA_FEATURE=EQN_TEXT_ENTRY,ADDITIONAL_GROUP_MANAGERS,HIGH_CONTRAST,SCHEDULE_ASS
 EXAM_BOARD=AQA,OCR
 DISPLAY_SETTING=HIDE_QUESTION_ATTEMPTS,HIDE_NON_AUDIENCE_CONTENT
 BOOLEAN_NOTATION=ENG,MATH
+
+# Google reCAPTCHA
+GOOGLE_RECAPTCHA_SECRET=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
```

Comment:
Not a big deal, but the reason why this red circle is appearing is that this file (and the other one) doesn't end with a line break. You can configure your IDE to ensure that every file ends with a line break on save.

Reply:
Ah yes sorry my intellij has nothing configured because I usually only use it for running the API and don't tend to make any code changes!